index.js

```diff
@@ -678,6 +680,10 @@ class Readable extends Stream {
     return (rs._duplexState & READ_BACKPRESSURE_STATUS) !== 0 || rs._readableState.buffered >= rs._readableState.highWaterMark
   }
 
+  static isPaused (rs) {
+    return (rs._duplexState & READ_PAUSED) !== 0 && (rs._duplexState & READ_RESUMED) === 0
```

Comment:
this first part here is redundant, only `(rs._duplexState & READ_RESUMED) === 0` is needed

Reply:
Gotcha! Fixed in recent commit.